draft-sheffer-oauth-rfc8725bis.md

```diff
@@ -467,16 +472,15 @@ without new algorithm identifiers being required.
 
 ## Validate All Cryptographic Operations {#validate-crypto}
 
-
- All cryptographic operations used in the JWT  MUST be
- validated and the entire JWT  MUST be rejected
+All cryptographic operations used in the JWT MUST be validated and the entire JWT MUST be rejected
 if any of them fail to validate.
 This is true not only of JWTs with a single set of Header Parameters
 but also for Nested JWTs in which both outer and inner operations  MUST be validated
 using the keys and algorithms supplied by the application.
 
+Libraries MUST allow the verifier to distingush between JWS-signed and JWE-encrypted JWTs, and to only accept the former.
```

Comment:
```suggestion
Libraries MUST allow the verifier to distinguish between JWS-signed and JWE-encrypted JWTs, enabling callers to accept only JWTs that meet their security criteria.
```

Reply:
That would allow the verifier to only accept JWS, if that's their policy.